### ISSUE TITLE
error AL0185: table 'XXXX' is missing in extension

### ISSUE THREAD

**Issue by dbsae** [microsoft/AL#5905]
**Describe the bug**
We have one parent app in which we customized the Employee Screen and add lot of custom field. We are making other app depending of first app. after downloading the symbols custom fields are not appearing.  This happened after last update of Business central  16.1.12629.12805

**To Reproduce**
Steps and to reproduce the behavior:
1. Create an FirstApp Customize the Employee Table and add some custom field and publish in business central. 
2. Create and Second app and add first app in this app. When you trying to access the employee table you cannot able to find those custom fields.

``` AL Code to reproduce the issue ```

**Expected behavior**

**Screenshots**

**5. Versions:**

 - 5.0.271898

 - Business Central:

**Comment by atoader** [comment]
Can you please provide the code to reproduce this issue?

**Comment by dbsae** [comment]
Hi Alexandru Toader,

As requested find the AL code to reproduce the bug. Please include the below libraries we have removed because size was to big to upload here.

Microsoft_Base Application_16.1.12629.13235.app
Microsoft_System Application_16.1.12629.13130.app
Microsoft_System_16.0.12630.12758.app

In First App we just extend the employee table and create one field. and in second app when we are include the first app as dependencies and trying to use this field but we could not able to find the field.

[FirstApp.zip](https://github.com/microsoft/AL/files/4663915/FirstApp.zip)
[SeccondApp.zip](https://github.com/microsoft/AL/files/4663916/SeccondApp.zip)

**Comment by dbsae** [comment]
Dear Alexandru,

We have uploaded the al code to reproduce the issue.

Regards,

Noman Mansoor  Technical Lead / CRM Techno Functional Consultant
Dynamics Business Solutions (DBS)
2903 HDS Business Center, JLT, PO Box 336613, Dubai, UAE
Tel: +971 4 447 5525  |  Mobile: +971 50 7633794
noman@dbsae.com<mailto:elmaraghy@dbsae.com>  | www.dbsae.com<http://www.dbsae.com/>
P please don't print this e-mail unless you really need to.

**Comment by atoader** [comment]
Thank you! We will take a look.

**Comment by dbsae** [comment]
Hi Sir,

Have you get a chance to look at this issue. We are need to resolve this issue because as you know we cannot extend from base app as dependencies. Waiting for your reply.


Thanks

**Comment by atoader** [comment]
Hi! It is on our backlog, but I cannot provide an ETA. We try to address the issues filed on GitHub on as much as our capacity allows, but issues raised through support or issues concerning the online SaaS service will get a higher priority.

If the issue is urgent, I recommend opening a support case.

- Open a support request to CSS through the PartnerSource portal
- Contact your Service Account Manager (SAM) in your local subsidiary to understand what is included in your contract as of support incident and PAH (Partner Advisory Hours). Your SAM might also direct you "step by step" how to open a support request or how to get credentials, if this is the first time for you or your company

If you file the bug in Collaborate, remember to include steps to reproduce the issue, and the Business Central build number and country version you're using.

**Comment by AndreyKorepanov** [comment]
Reproducible on the sample apps attached

**Comment by qutreson** [comment]
Hi @dbsae are you still experiencing this issue in our latest developer preview?

I couldn't reproduce this issue on our latest builds. Here are the steps I followed:
1. Open FirstApp in VS Code
2. Build and publish FirstApp from VS Code
3. Open SecondApp in VS Code
4. Download the reference symbols for SecondApp (to satisfy the dependency on FirstApp)
5. Build and publish SecondApp from VS Code

I also tried moving the two extensions in the same workspace. 

I am not getting any errors, just a warning about the use of implicit rec and the extension is successfully published.

![image](https://user-images.githubusercontent.com/31541422/93076469-ae174580-f687-11ea-87e8-aa9f21471125.png)


![image](https://user-images.githubusercontent.com/31541422/93076434-9f309300-f687-11ea-854f-a18f5d931141.png)
